chisel/Cargo.toml

```diff
@@ -22,6 +22,10 @@ path = "src/cli.rs"
 [dependencies]
 foundry-evm = { path = "../evm" }
 forge = { path = "../forge" }
+foundry-config = { path = "../config" }
+foundry-cli = { path = "../cli" }
```

Comment:
do we want another standalone binary, or should this rather be a forge subcommand @gakonst ?

Reply:
Either one works! Let me know what your guys' preference is, can change things around if need be.

Reply:
ii like the idea of a new binary :)

Reply:
> ii like the idea of a new binary as long as it works even w/o being inside a foundry project

Sounds good! This is the plan- will work with default configuration / any supplied flags if outside of a foundry project. Was planning on adding certain features that are enabled if the pwd is a foundry project if that's okay, though:
* Auto-importing project interfaces.
* `foundry.toml` configuration support via the [`LoadConfig`](https://github.com/foundry-rs/foundry/blob/master/cli/src/cmd/utils.rs#L196-L222) trait.
* Support for project remappings wrt manual imports.
* Support for RPC mappings via `foundry.toml`'s `[rpc_endpoints]` section. (i.e. `!fork mainnet`)
* Exporting session source to a script within the project's `scripts` dir.